.eslintrc.js

```diff
@@ -11,6 +14,46 @@ module.exports = {
     'src/govuk/vendor/polyfills/**/*'
   ],
   overrides: [
+    {
+      extends: 'plugin:jsdoc/recommended',
+      files: ['**/*.{cjs,js,mjs}'],
+      plugins: [
+        'jsdoc'
+      ],
+      rules: {
+        // JSDoc blocks are optional
+        'jsdoc/require-jsdoc': 'off',
+        'jsdoc/require-param-description': 'off',
+        'jsdoc/require-param': 'off',
+
+        // Require hyphens before param description
+        // Aligns with TSDoc style: https://tsdoc.org/pages/tags/param/
+        'jsdoc/require-hyphen-before-param-description': 'warn',
```

Comment:
We sometimes hyphenate, sometimes don't

It's [recommended by JSDoc](https://jsdoc.app/tags-param.html) (and mandatory for TSDoc)

>If you provide a description, you can make the JSDoc comment more readable by inserting a hyphen before the description. Be sure to include a space before and after the hyphen.